src/datum_sockets.c

```diff
@@ -283,11 +283,17 @@ void *datum_threadpool_thread(void *arg) {
 										DLOG_DEBUG("New proxy IP detected: %s on CID: %d, TID: %d", src_ip, my->client_data[cidx].cid, my->thread_id);
 										strncpy(my->client_data[cidx].rem_host, src_ip, DATUM_MAX_IP_LEN-1);
 									} else {
-										DLOG_DEBUG("PROXY line present but no valid IP found, keeping original source IP: %s", my->client_data[cidx].rem_host);
+										DLOG_DEBUG("PROXY line present but no valid IP found, keeping original source IP: %s on CID: %d, TID: %d", my->client_data[cidx].rem_host,  my->client_data[cidx].cid, my->thread_id);
 									}
 								}
 								start_line = end_line + 1;
 								end_line = strchr(start_line, '\n');
+								
+								if (start_line[0] == '{') {
```

Comment:
Probably should just have this be anything that doesn't fit the PROXY line template.